README.md

```diff
@@ -167,6 +167,7 @@ To build using gradle, there are a number of required dependencies:
   - Be sure to set the `ANDROID_HOME` environment variable
 - [Android Command-line tools](https://developer.android.com/studio#command-line-tools-only)
   - Be sure to install at the correct location so binaries are located at `$ANDROID_HOME/cmdline-tools/<VERSION>/bin/`
+  - Before building, you must invoke `sdkmanager --licenses` to accept the licenses
```

Comment:
there is no reference about using `sdkmanager` in the readme so i think this doesn't make sense here

Reply:
```suggestion
  - Please note, the command-line tools include the `sdkmanager` binary, which is used during the build.  Before building, you must accept the sdkmanager licenses by invoking `sdkmanager --licenses`.
```

is this sufficient?